src/components/pages/about_us/AboutUsContainer.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import AboutUsView from './AboutUsView';
+
+/**
+ * About us page container.
+ */
+const AboutUsContainer = () => (
+  <AboutUsView />
```

Comment:
It's the container-view pattern. State managment and event handling in the container, and view in the view. Rudimentary version of MVC. Seems a little silly now because there is no state management but just future proofing